cmd/package.go

```diff
@@ -558,8 +570,17 @@ func DescribePackage(repo *util.Repo, packageName string) error {
 			}
 		}
 
+		// Read meta/README.md
+		if strings.HasSuffix(header.Name, "meta/README.md") {
```

Comment:
I know it's just replicating previous pattern, but currently we always check all `if`s. Adding `continue` or using `else if` would probably make sense.